src/python/pants/jvm/resources.py

```diff
@@ -89,9 +91,21 @@ async def assemble_resources_jar(
         ProcessResult,
         Process(
             argv=[
-                zip.path,
-                output_filename,
-                *sorted(input_files),
+                bash.path,
+                "-c",
+                " ".join(
+                    [
+                        touch.path,
+                        "-d 1980-01-01T00:00:00Z",
+                        *sorted(input_files),
+                        ";",
```

Comment:
```suggestion
                        "&&",
```
(otherwise `touch` might fail while the rest of the command succeeds.)